<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZmwAnalysisWidget</class>
 <widget class="QMainWindow" name="ZmwAnalysisWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="ImageView" name="imagePlot">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>930</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCrop"/>
    <addaction name="actionFilter"/>
    <addaction name="actionClear_Settings"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZ_Project"/>
    <addaction name="actionView_Stack"/>
   </widget>
   <widget class="QMenu" name="menuAnalyze">
    <property name="title">
     <string>Analyze</string>
    </property>
    <addaction name="actionBase_Call"/>
    <addaction name="actionCheck_Controls"/>
    <addaction name="actionSet_Threshold"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuAnalyze"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad">
   <property name="text">
    <string>load</string>
   </property>
  </action>
  <action name="actionCrop">
   <property name="text">
    <string>Crop</string>
   </property>
  </action>
  <action name="actionFilter">
   <property name="text">
    <string>Filter</string>
   </property>
  </action>
  <action name="actionZ_Project">
   <property name="text">
    <string>Z-Project</string>
   </property>
  </action>
  <action name="actionView_Stack">
   <property name="text">
    <string>View Stack</string>
   </property>
  </action>
  <action name="actionBase_Call">
   <property name="text">
    <string>Base Call</string>
   </property>
  </action>
  <action name="actionCheck_Controls">
   <property name="text">
    <string>Check Controls</string>
   </property>
  </action>
  <action name="actionSet_Threshold">
   <property name="text">
    <string>Set Threshold</string>
   </property>
  </action>
  <action name="actionClear_Settings">
   <property name="text">
    <string>Clear Settings</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageView</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
